### ISSUE TITLE
Keycloak redirect_uri: when going to /auth or authorizing using swagger, a keycloak error page is shown

### ISSUE THREAD

**Issue by DonWillems** [openremote/openremote#814]
When going to the keycloak administration console, or when trying to authorize with a service user in Swagger, you end up on a page that shows the OpenRemote logo, 'We are sorry', and 'Invalid parameter: redirect_uri'
Additionally when coming from the /auth page, it shows a button to go back: '[« Back to Application](https://staging.demo.openremote.app/auth/admin/master/console/)'
When you press that and log in, you get redirected to the correct page.

Trying to access e.g. staging.demo.openremote.app/auth/ and then the administration console. Gives this URL:
`https://staging.demo.openremote.app/auth/realms/master/protocol/openid-connect/auth?client_id=security-admin-console&redirect_uri=https%3A%2F%2Fstaging.demo.openremote.io%2Fauth%2Fadmin%2Fmaster%2Fconsole%2F&state=6cde4882-6c87-42-8a0d-ed8f2a1e8f&response_mode=fragment&response_type=code&scope=openid&nonce=f0d4ac-f2c0-405b-ab43-0371d6b4f4e5&code_challenge=JUI6w7UsyzsbsAkffepAtw81HrKfDALaN_qCN_ZG8&code_challenge_method=S256`

Coming from Swagger authorize:
`https://staging.demo.openremote.app/auth/realms/master/protocol/openid-connect/auth?response_type=code&client_id=testtest&redirect_uri=https%3A%2F%2Fstaging.demo.openremote.app%2Fswagger%2Foauth2-redirect.html&state=TW9uIFNlcCAyNiAyMDIyIOjA1OjM3IEdNVCswMjAwIChDZW50cmFsIEV1cm9wZWFuIFN1bW1lciBUaW1lKQ%3D%3D`

**Comment by bryanhonof** [comment]
I am experiencing the same behaviour. I have noticed, when entering `openremote` as `client_id` and leave the `client_secret` empty, the flow *seems* to be fine. But I'm uncertain if this is desired behaviour.

**Comment by richturner** [comment]
Yes you just accept the defaults when authorizing...swagger UI is not great at handling OAuth flows for example it doesn't handle token refresh or realm placeholders in the token URI...need to check if things have improved at all.
